pkgs/development/libraries/libvorbis/default.nix

```diff
@@ -1,28 +1,15 @@
-{ stdenv, fetchurl, libogg, pkgconfig, fetchpatch }:
+{ stdenv, fetchurl, libogg, pkgconfig }:
 
 stdenv.mkDerivation rec {
-  name = "libvorbis-1.3.6";
+  name = "libvorbis-1.3.7";
```

Comment:
what about pname and version?

Reply:
Its a backport so better to minimize the changes.